seed/data_importer/meters_parser.py

```diff
@@ -5,13 +5,12 @@
 
```

Comment:
This file got completely revamped, so GH has a hard time showing changes. It might be best to review the resulting file vs the differences.